src/components/home/UpcomingCard.jsx

```diff
@@ -1,3 +1,30 @@
-const UpcomingCard = () => {};
+const UpcomingCard = ({ month, date, time, event, location, description }) => {
```

Comment:
```suggestion
const UpcomingCard = ({ month = "FEB" , date = "19" , time, event, location, description }) => {
```

instead of default props at the bottom you can do this instead.